src/Controls/tests/Core.UnitTests/TestClasses/AnimationReadyHandler.cs

```diff
@@ -83,7 +89,12 @@ public object GetService(Type serviceType)
 				if (serviceType == typeof(IAnimationManager))
 					return _animationManager;
 
-				throw new NotSupportedException();
+				if (serviceType == typeof(IDispatcherProvider))
+				{
+					return DispatcherProvider.Current;
+				}
```

Comment:
What was this provider used for in the new code?

Reply:
Before I wrote all the infrastructure to force the tests to run on a single thread, it was added so the tests wouldn't throw a NotSupportedException when something needed a dispatcher. It's probably not be necessary anymore.